### DISCUSSION TITLE
Make an option to disable 3-D building

### DISCUSSION THREAD

**Discussion by hoizan** [streetcomplete/StreetComplete#5235] [Feedback]
The 3-D buildings images can be quite irritating as they often obstruct the editing. Alternatively, make it able to lower the opacity of the 3-D images.

**Comment by mnalis** [comment]
- I agree, but was unable to convince powers-that-be at the time: see https://github.com/streetcomplete/StreetComplete/issues/2206#issuecomment-719071926

Workaround is to  enable `Address overlay` (see https://github.com/streetcomplete/StreetComplete/issues/2206#issuecomment-1276657156) (or alternatively to use [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) fork instead, which has preference for `Show 3D buildings`)


Perhaps you can try to produce more convincing arguments then what I mustered, or maybe it would be easier to do so now (as the   automatic disabling of 3D in Address overlay probably means that the problem was recognized _at least_ in some situations)

**Comment by hoizan** [comment]
Hello, Thanks for your reply and pointing me to the existing case. Sorry
for this duplication. However, the workaround by switching to Address
overlay is awful because I primary doing Pavement and street related
surveys. I guess lower the opacity of the 3-D objects is good enough though.

Anyway, thank you very much for your response.

**Reply by mnalis** [reply]
Sure, no worries. When you feel you've gathered enough feedback in this Discussion, feel free to open the [new Issue (Feature Request type)](https://github.com/streetcomplete/StreetComplete/issues/new?assignees=&labels=&projects=&template=feature_request.md) and fill in information requested in the template. 

That will push that idea to project owner for consideration/implementation.
